common/src/main/scala/lucuma/react/common/jsComponents.scala

```diff
@@ -26,18 +26,48 @@ class AttrsBuilder(p: js.Object) extends VdomBuilder.ToJs {
 
 trait GenericFnComponent[P <: js.Object, CT[-p, +u] <: CtorType[p, u], U] {
   protected def cprops: P
-  @inline def render: RenderFn[P]
+  def render: RenderFn[P]
 }
 
+object GenericFnComponent:
+  given [P <: js.Object, CT[-p, +u] <: CtorType[p, u], U]
+    : Conversion[GenericFnComponent[P, CT, U], RenderFn[P]] =
+    _.render
+
+  given [P <: js.Object, CT[-p, +u] <: CtorType[p, u], U]
+    : Conversion[GenericFnComponent[P, CT, U], VdomNode] =
+    _.render
+
+  given [P <: js.Object, CT[-p, +u] <: CtorType[p, u], U]
+    : Conversion[GenericFnComponent[P, CT, U], js.UndefOr[VdomNode]] =
+    _.render
+
 trait GenericFnComponentC[P <: js.Object, CT[-p, +u] <: CtorType[p, u], U, A] {
   protected def cprops: P
   val children: CtorType.ChildrenArgs
   def withChildren(children: CtorType.ChildrenArgs): A
 
-  @inline def renderWith: RenderFnC[P]
-  @inline def render: RenderFn[P] = renderWith(children)
+  def renderWith: RenderFnC[P]
+  def render: RenderFn[P] = renderWith(children)
```

Comment:
Did you want these, and the one on line 29, to be `inline` like the rest of the `render` defs in the file?

Reply:
The compiler emitted a warning that `inline` there may degrade performance, so I removed it.